README.md

```diff
@@ -139,6 +139,11 @@ clj -M:clojure-lsp diagnostics
 - [pg-embedded-clj](https://github.com/Bigsy/pg-embedded-clj) Embedded PostgreSQL for integration tests
 - [clojure-lsp](https://github.com/clojure-lsp/clojure-lsp/) Code Format, Namespace Check and Diagnosis
 
+# Documentation
+
+If you want to know more about the components in general and each component or helper implementation, check the [documentation](doc/intro.md).
```

Comment:
I would list all the possible documents somewhere. Maybe here or an "index" readme.md on the root of the doc folder.

Reply:
Clojars has an auto documentation website, if you follow their specs.

And reading it I would do the list in the main readme and not create a doc/readme.md

https://github.com/cljdoc/cljdoc/blob/master/doc/userguide/for-library-authors.adoc

There is an file name thing for ordering the articles, and some other configs that may interest us.

Reply:
There this compile error on cljdoc

https://app.circleci.com/pipelines/github/cljdoc/builder/56926/workflows/358c1aa3-f76e-4059-9bba-7bdbaf1aa154/jobs/73301/parallel-runs/0/steps/0-105

Maybe we will need to add state flow on the main deps

Reply:
https://github.com/cljdoc/cljdoc/issues/951